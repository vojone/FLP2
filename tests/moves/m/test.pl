:- consult("../../../cube").

:- read_cube(C, _), move_m_cw(C, _, _, RC), write_cube(RC).
