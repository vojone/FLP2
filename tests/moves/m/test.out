133
436
729
424 111 232 333
151 222 323 444
181 222 313 444
646
616
616
